Lib/test/test_strptime.py

```diff
@@ -354,7 +356,7 @@ def test_julian(self):
         # Test julian directives
         self.helper('j', 7)
 
-    def test_offset(self):
+    def test_z_directive_offset(self):
```

Comment:
Not sure if this rename is necessary, considering the the surrounding code's method naming convention.